barcode_utils.py

```diff
@@ -0,0 +1,166 @@
+from barcode import *
+
+import base64
+import struct
+import skip32
+import code128
+import os
+
+
+# TODO: currently broken, need to rewrite in the context of uber
+def generate_barcode_csv(filename):
+    badge_types = self.badge_types['badge_types']
+
+    # TODO: modify so we read the config directly from uber.
+    lines = []
+    for badge_type, ranges in badge_types.items():
+        range_start = int(ranges['range_start'])
+        range_end = int(ranges['range_end'])
+        lines = lines + self.generate_barcode_nums(range_start, range_end)
+
+    f = open(filename,'w')
+
+    for line in lines:
+        f.write(line + os.linesep)
+
+    f.close()
+
```

Comment:
Yeah, I would prefer web access in general, but in this case a sep command seems like the best solution until we have better admin privileges.